Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,14 +1,40 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.IO;
-using Console = System.Console;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
         static void Main(string[] args)
```

Comment:
Move all from main to Lesson3 as mentioned in homework assignment.

Reply:
> Move all from main to Lesson3 as mentioned in homework assignment.

Moved program logic to class Lesson3